packages/gallery/src/components/item/videos/videoItemWrapper.js

```diff
@@ -64,11 +65,8 @@ class VideoItemWrapper extends ImageItem {
       'settings',
       'actions',
     ]);
-    const shouldCreatePlaceHolder =
-      utils.isSingleItemHorizontalDisplay(styleParams) && 
-      styleParams.videoPlay === GALLERY_CONSTS.videoPlay.AUTO;
 
-    return shouldCreatePlaceHolder ? null : (
+    return !shouldCreateVideoPlaceholder(styleParams) ? null : (
```

Comment:
Same here. I think I was supposed to dig in a bit more into this and give you an alternative approach to implement this. Maybe we should dig in to it together.